src/Database/DatabaseControllerKubeOps/Controller/Controllers/DataCtrl.cs

```diff
@@ -0,0 +1,141 @@
+﻿
+using KubeOps.Operator.Controller;
+using KubeOps.Operator.Rbac;
+using DatabaseControllerKubeOps.Controller.Entities;
+using KubeOps.Operator.Controller.Results;
+using k8s;
+using k8s.Models;
+using System.Text.Json;
+using System.Threading;
+using Grpc.Core;
+using Grpc.Net.Client;
+using GrpcDatabaseService;
+using Microsoft.Rest;
+
+namespace DatabaseControllerKubeOps.Controller.Controllers;
+
+
+internal class OnChange
+{
+
+    public static async void UpdateDatabase(V1Alpha1DataEntity entity)
+    {
+        try
+        {
+            var values = new Dictionary<string, string>
+            {
+                { "data", JsonSerializer.Serialize(entity) },
+            };
+            var content = new FormUrlEncodedContent(values);
+
+            HttpClient httpClient = new HttpClient();
+
+            var response = await httpClient.PostAsync("http://database-sync-service:3000/listen-for-database-schema", content);
+            var responseString = await response.Content.ReadAsStringAsync();
+            Console.WriteLine(responseString);
+        }
+        catch (HttpOperationException httpOperationException) when (httpOperationException.Message.Contains("422"))
+        {
+            var phase = httpOperationException.Response.ReasonPhrase;
+            var content = httpOperationException.Response.Content;
+            Console.WriteLine("response content: {0}", content);
```

Comment:
I found that you can't dependency inject directly into the DataCtrl class because it doesn't implement IHostedService, so I created a class public class Logger<T> : IHostedService, ILogger<T>
that implements IHOsted Service and ILogger so I can dependency inject logger, is that how you would do it?

Reply:
Actually I'm thinking that ILogger gets auto dependency injected by dotnet, so I won't do this.